src/analysis/normed_space/basic.lean

```diff
@@ -282,6 +279,16 @@ export norm_one_class (norm_one)
 @[simp] lemma nnnorm_mul (a b : α) : ∥a * b∥₊ = ∥a∥₊ * ∥b∥₊ :=
 nnreal.eq $ norm_mul a b
 
+lemma list.norm_prod : ∀ l : list α, ∥l.prod∥ = (l.map norm).prod
+| [] := by simp
+| (a :: l) :=
+  by rw [list.map_cons, list.prod_cons, @list.prod_cons _ _ _ ∥a∥, norm_mul, list.norm_prod]
+
+lemma list.nnnorm_prod : ∀ l : list α, ∥l.prod∥₊ = (l.map nnnorm).prod
+| [] := by simp
+| (a :: l) :=
+  by rw [list.map_cons, list.prod_cons, @list.prod_cons _ _ _ ∥a∥₊, nnnorm_mul, list.nnnorm_prod]
```

Comment:
Can you prove this with `(norm_hom.to_monoid_hom : α →* ℝ).map_list_prod f l` or similar?

Reply:
ah, nice. Yes, that works.